airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/StreamDescriptorUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination_async;
+
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
+import io.airbyte.protocol.models.v0.AirbyteStream;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream;
+import io.airbyte.protocol.models.v0.StreamDescriptor;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Boring utility functions that I don't want in my face.
```

Comment:
lol. i wrote this javadoc comment. we can probably do better. 😆

Reply:
made slightly better :)